views/header.ejs

```diff
@@ -0,0 +1,21 @@
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>jenkins-wiki-exporter - Quick tool to convert your plugin docs from wiki to github markdown.</title>
+    <meta name="description" content="Quick tool to convert your plugin docs from wiki to github markdown.">
+    <meta name="author" content="Gavin Mogan">
```

Comment:
We can probably remove these metatags, i copy and pasted them from an earlier project